drivers/hmis/app/jobs/hmis/migrate_assessments_job.rb

```diff
@@ -43,7 +43,7 @@ class MigrateAssessmentsJob < BaseJob
     #  - DateCreated = earliest creation date of related records
     #  - DateUpdated = latest update date of related records
     #  - UserID = UserID from the related record that was most recently updated
-    def perform(data_source_id:, project_ids: nil, clobber: false, delete_dangling_records: false, preferred_source_hash: nil, generate_empty_intakes: false)
+    def perform(data_source_id:, project_ids: nil, clobber: false, delete_dangling_records: false, preferred_source_hash: nil, generate_empty_intakes: false, enrollments: nil)
```

Comment:
suggest adding some documentation for these params. Also, is there a case for passing both project_ids and enrollment_ids? Seems like they might conflict. Perhaps should raise an ArgumentError if both are supplied
```suggestion
    # - enrollments = active record enrollment scope, only create records for enrollments in this scope; default is all enrollments at project_ids
    # - project_ids = array of integers, project pks, only create records for enrollments at these projects in the same data source
    def perform(data_source_id:, project_ids: nil, clobber: false, delete_dangling_records: false, preferred_source_hash: nil, generate_empty_intakes: false, enrollments: nil)
```

Reply:
Added argument error and param descriptions, thanks!